### DISCUSSION TITLE
Publish Repo to GitHub Team/Organization on GitHub Desktop

### DISCUSSION THREAD

**Discussion by benehudson** [community/community#22922] [New to GitHub]
<p>I’m new to git/github.  Using the VS 2017 GitHub extension I was able to publish my first private repo to my GitHub team/organization.  Woohoo!</p>
<p>I’ve got some pre VS 2015 projects that I need to publish to GitHub.  For that I’ve downloaded the GitHub desktop applicaiton.  I’ve logged into my personal GitHub account in the desktop app, and I can create repos there all day.  But, what I want to do, and can’t figure out how is to create private repos in my team/organization using the desktop application.</p>
<p>Can someone outline how that’s done, or point me to a tutorial online?</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/benehudson">@benehudson</a> we have a help guide for how to publish a repository to GitHub using GitHub Desktop here:</p>
<p><a href="https://help.github.com/desktop/guides/contributing-to-projects/adding-an-existing-project-to-github-using-github-desktop/" rel="nofollow noopener">https://help.github.com/desktop/guides/contributing-to-projects/adding-an-existing-project-to-github-using-github-desktop/</a></p>
<p>If you have any specific questions about this process, please let me know!</p>
